src/microstrain_inertial_rqt/rtk_status.py

```diff
@@ -28,26 +29,62 @@ def run(self):
       self.rtk_widget.show()
 
     # Update device specific data
-    self._update_rtk_data()
+    if self._rtk_status_monitor.version == 1:
+      self.rtk_v2_widget.show()
+      self.rtk_v1_widget.hide()
+
+      self._update_rtk_data_v2()
+    else:
+      self.rtk_v1_widget.show()
+      self.rtk_v2_widget.hide()
+
+      self._update_rtk_data_v1()
+    
+  def _update_rtk_data_v1(self):
+    # V1 Status Flags
+    self.rtk_status_flags_mode_label.setText(self._rtk_status_monitor_v1.controller_state_string)
+    self.rtk_status_flags_controller_status_label.setText(self._rtk_status_monitor_v1.controller_status_string)
+    self.rtk_status_flags_device_state_label.setText(self._rtk_status_monitor_v1.platform_state_string)
+    self.rtk_status_flags_connection_status_label.setText(self._rtk_status_monitor_v1.platform_status_string)
+    self.rtk_status_flags_reset_reason_label.setText(self._rtk_status_monitor_v1.reset_reason_string)
+
+    # V1 Controller state icon label
+    self.rtk_led_status_label.setText(self._rtk_status_monitor_v1.controller_status_string)
+
+    # Update common flags
+    self._update_rtk_data(self._rtk_status_monitor_v1)
     
-  def _update_rtk_data(self):
+  def _update_rtk_data_v2(self):
+    # V2 Status Flags
+    self.rtk_status_flags_modem_state_label.setText(self._rtk_status_monitor.modem_state_string)
+    self.rtk_status_flags_connection_type_label.setText(self._rtk_status_monitor.connection_type_string)
+    self.rtk_status_flags_rssi_label.setText(self._rtk_status_monitor.rssi_string)
+    self.rtk_status_flags_tower_change_indicator_label.setText(self._rtk_status_monitor.tower_change_indicator_string)
+    self.rtk_status_flags_nmea_timeout_label.setText(self._rtk_status_monitor.nmea_timeout_string)
+    self.rtk_status_flags_server_timeout_label.setText(self._rtk_status_monitor.server_timeout_string)
+    self.rtk_status_flags_rtcm_timeout_label.setText(self._rtk_status_monitor.rtcm_timeout_string)
+    self.rtk_status_flags_out_of_range_label.setText(self._rtk_status_monitor.out_of_range_string)
+    self.rtk_status_flags_corrections_unavailable_label.setText(self._rtk_status_monitor.corrections_unavailable_string)
+
+    # V1 Modem state icon label
+    self.rtk_led_status_label.setText(self._rtk_status_monitor.modem_state_string)
+
+    # Update common flags
+    self._update_rtk_data(self._rtk_status_monitor)
+
+  def _update_rtk_data(self, rtk_monitor: RTKMonitorBase):
```

Comment:
Unfortunately since we support melodic, we cannot use typehints as melodic uses python2. This should be removed and the import for `RTKMonitorBase` could also be removed